.travis.yml

```diff
@@ -14,56 +13,24 @@ addons:
   #     - postgresql-9.6-postgis-2.3
 
 python:
-  - "2.7"
-  - "3.4"
-  - "3.5"
   - "3.6"
   - "3.7"
+  - "3.8"
 
 env:
-  - DJANGO_VERSION='Django>=1.8,<1.9'
-  - DJANGO_VERSION='Django>=1.9,<1.10'
-  - DJANGO_VERSION='Django>=1.10,<1.11'
-  - DJANGO_VERSION='Django>=1.11,<2.0'
-  - DJANGO_VERSION='Django>=2.0,<2.1'
-  - DJANGO_VERSION='Django>=2.1,<2.2'
+  - DJANGO_VERSION='Django>=1.11,<1.12'
```

Comment:
This will create a 3x3 build matrix, but we should either exclude Django 1.11 with Python 3.8, or we should allow that build to fail. My vote is the second one.